streams/src/main/java/org/apache/kafka/streams/processor/WallclockTimestampExtractor.java

```diff
@@ -28,6 +28,13 @@
  * built-in <i>CreateTime</i> timestamp (see KIP-32: Add timestamps to Kafka message for details).
```

Comment:
@guozhangwang: After talking to @ijuma I realized that `ConsumerRecordTimestampExtractor` (to which we refer here) is actually not always returning the CreateTime timestamp (= producer time) -- it could also be LogAppendTime (broker time), and which of the two you'd get depends on the topic-level configuration, right?